src/components/ViewerBorder/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './ViewerBorder.scss';
+
+
+const ViewerBorder = props => (
+  <span id={props.spanId}>
+    <div className={`viewer-border${props.spanFullWidth ? ' w-100' : ''}`}>
+      { (props.title.trim() !== '') ?
```

Comment:
this would read easier just using and (`&&`) rather than the tertiary

props.title.trim() && <div ...>

If there is no title it won't output the div, which is what I think you were going for.  This applies down below too.

Reply:
Thanks! I didn't realize that was an option!